ls1intum/Artemis
# `Development`: Rename "Athene" to "Athena"

### Checklist
#### General


#### Server
None of the checks apply, I just renamed parts of the code
#### Client
None of the checks apply, I just renamed parts of the code

### Motivation and Context


Historically, both names, "Athene" and "Athena" have been used for the semi-automatic (text) assessment system.
We want to be consistent about this. Therefore, we decided to always use the name "Athena" (same as the repository name: https://github.com/ls1intum/Athena) from now on.

As part of migrating the current connection between Artemis and Athena to the new system already found at https://github.com/ls1intum/Athena (#6861), I am also doing a renaming. This is a separate PR so that the actual integration PR will be easier to review.

### Description

I renamed every instance of code with the name "athene" in it to "athena". This includes run configurations. I submitted a PR to https://github.com/ls1intum/artemis-ansible-collection/pull/42 covering this as well.

The only place where the name `athene` is still used is the `de.tum.in.ase.athene` gradle package, which I don't want to rename as it will be removed in a future PR anyways. Therefore, some imports like `import de.tum.in.ase.athene.protobuf.Cluster;` are still using the old name as well.

### Steps for Testing

Unfortunately, you cannot really test the functionality of this PR, as the old Athena system is currently down.
As long as nothing breaks, this is fine, though. In a future PR, the new connection to Athena will be tested anyways and until then, everything remains as-is anyways.

Still, you can
1. Try to find remaining places to rename
2. Deploy this PR to a test server and verify that it generally works
3. Run this PR locally using the `Artemis (Server, Athena)` run configuration to verify that this still generally works as well

### Review Progress






#### Code Review

### Test Coverage






unchanged (only renaming)
